Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Networking.sb.in

```diff
@@ -374,7 +374,8 @@
 ))
 
 (define required-etc-files
-  (literal "/private/etc/hosts"
+  (literal "/private/etc/group"
```

Comment:
I think we should also consider whether we would like to silence this, since it doesn't seem necessary for a Networking process to have knowledge about group information.

Reply:
That's a good point. I looked through `libinfo` use cases, and can't see where it's accessed in a way that's relevant to WebKit uses. I'll revise this patch to silence the warning.

I did an A/B performance test to see if allowing it changed anything, and there were no improvements so there doesn't seem to be any value in unblocking.

Reply:
Thank you!